gui/mainwindow.cpp

```diff
@@ -79,11 +79,20 @@ MainWindow::MainWindow(TranslationHandler* th, QSettings* settings) :
     mLineEditFilter = new QLineEdit(mUI.mToolBarFilter);
     mLineEditFilter->setPlaceholderText(tr("Quick Filter:"));
     mUI.mToolBarFilter->addWidget(mLineEditFilter);
-    connect(mLineEditFilter, SIGNAL(textChanged(const QString&)), mFilterTimer, SLOT(start()));
+#ifndef QOVERLOAD_FALLBACK
+    connect(mLineEditFilter, &QLineEdit::textChanged, mFilterTimer, QOverload<>::of(&QTimer::start));
+#else
+    connect(mLineEditFilter, &QLineEdit::textChanged, mFilterTimer, static_cast<void (QTimer::*)(void)>(&QTimer::start));
+#endif
+
     connect(mLineEditFilter, &QLineEdit::returnPressed, this, &MainWindow::filterResults);
 
-    connect(mUI.mActionPrint, SIGNAL(triggered()), mUI.mResults, SLOT(print()));
-    connect(mUI.mActionPrintPreview, SIGNAL(triggered()), mUI.mResults, SLOT(printPreview()));
+#ifndef QOVERLOAD_FALLBACK
+    connect(mUI.mActionPrint, &QAction::triggered, mUI.mResults, QOverload<>::of(&ResultsView::print));
```

Comment:
I suggest you use lambdas on 3-4 connects() and I can review that. if we are happy with the approach you can take care of the rest of the connects.